ORStools/proc/directions_points_layer_proc.py

```diff
@@ -218,6 +218,12 @@ def sort(f):
 
             try:
                 if optimization_mode is not None:
+                    # check for duplicate points
+                    if len(points) != len(set(points)):
+                        raise exceptions.DuplicateError(
+                            "There are duplicate points in the input layer, which need to be removed."
```

Comment:
```suggestion
                            "There are duplicate points in the input layer. Traveling Salesman Optimization does not allow this.
                            Either remove the duplicates or deselect Traveling Salesman.                         
```

Denke das machts noch bisschen klarer, was hier das Problem ist.